vault/data_source_aws_static_credentials_test.go

```diff
@@ -0,0 +1,55 @@
+package vault
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+	"testing"
+)
+
+func TestAccDataSourceAWSStaticCredentials(t *testing.T) {
+	_, _ = testutil.GetTestAWSCreds(t)
+	username := testutil.SkipTestEnvUnset(t, "AWS_STATIC_USER")[0]
+	mount := acctest.RandomWithPrefix("tf-aws-static")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testutil.TestAccPreCheck(t)
+			//SkipIfAPIVersionLT(t, testProvider.Meta(), provider.VaultVersion114)
```

Comment:
Should we enable this PreCheck?

Reply:
yeah, I missed re-including it after local tests